src/Recipient.c

```diff
@@ -251,6 +251,10 @@ bool _COSE_Recipient_decrypt(COSE_RecipientInfo * pRecip, COSE_RecipientInfo * p
 	int cbitKeyX = 0;
 	byte rgbKey[256 / 8];
 
+	UNUSED(rgbKey);
```

Comment:
This is confusing. Why would a local variable exist if it is unused. Probably need an #ifdef around the declaration instead.